app/ParserTests/ParserTests.hs

```diff
@@ -0,0 +1,117 @@
+{-|
+Description: Test Course Requirement Parsers using HUnit Testing Framework.
+
+Module containing test cases for Requirement Parsers.
+
+-}
+-- Example.hs  --  Examples from HUnit user's guide
+--
+-- For more examples, check out the tests directory.  It contains unit tests
+-- for HUnit.
+
+module ParserTests.ParserTests
+( runStrTests,
+  runReqTests
+) where
+
+import Test.HUnit
```

Comment:
@david-yz-liu I forgot to mention this but I made a judgement call to leave out the names as I found that I had to specify ~ 6. I realized I could use Test(..) instead of explicitly importing each value constructor and reduced it 4. Sorry for not communicating this, I pushed the changes as requested.